src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -143,6 +143,14 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $receiver = $this->receiverLocator->get($receiverName);
         $bus = $this->busLocator->get($busName);
 
+        $io = new SymfonyStyle($input, $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output);
+        $io->success(sprintf('Consuming messages from transport "%s" on bus "%s".', $receiverName, $busName));
+        $io->comment('Quit the server with CONTROL-C.');
```

Comment:
> Quit the worker with Control-C

Nice opportunity maybe to educate them that this is the layer that's known as the worker?

Also, it's minor, but we could also possibly send info about *when* it will quit automatically:

> The worker will automatically exit once it has (consumed 10 messages), (exceeded 128M of memory) or (has been running for 3600 seconds).
> Quit the worker at any time with Control-C

The `()` are added above as the conditionals - I think the English will sound ok if we figure out which 3 will apply, then basically implode on `,` with an extra `or` before the last one (if more than one). But, this might be getting a bit ridiculous :)

Reply:
Good idea, done.